### PR TITLE
Texture Inspector - Reverting clear

### PR DESCRIPTION
**Pull request by msDestiny14**
https://github.com/BabylonJS/Babylon.js/issues/9092

### CODE REVIEW

File: inspector/src/components/actionTabs/tabs/propertyGrids/materials/textures/textureEditorComponent.tsx [hunk 1]
```diff
@@ -313,7 +315,7 @@ export class TextureEditorComponent extends React.Component<ITextureEditorCompon
                 pickerOpen={this.state.pickerOpen}
                 setPickerOpen={this.setPickerOpen}
                 pickerRef={this._pickerRef}
-                hasAlpha={this.props.texture.hasAlpha}
+                hasAlpha={this.props.texture.textureFormat === -1 || this.props.texture.textureFormat === Constants.TEXTUREFORMAT_RGBA}
```
Comment (deltakosh) [inline inspector/src/components/actionTabs/tabs/propertyGrids/materials/textures/textureEditorComponent.tsx:318]: -1 means?

File: inspector/src/components/actionTabs/tabs/propertyGrids/materials/textures/textureEditorComponent.tsx [hunk 2]
```diff
@@ -16,6 +16,8 @@ import { PointerInfo } from 'babylonjs/Events/pointerEvents';
 
 import { PopupComponent } from '../../../../../popupComponent';
 import { ToolSettings } from './toolSettings';
+import { constants } from 'buffer';
```
Comment (deltakosh) [inline inspector/src/components/actionTabs/tabs/propertyGrids/materials/textures/textureEditorComponent.tsx:19]: why this line?

File: inspector/src/components/actionTabs/tabs/propertyGrids/materials/textures/textureEditorComponent.tsx (hunk 1 above)
Comment (msDestiny14) [reply inspector/src/components/actionTabs/tabs/propertyGrids/materials/textures/textureEditorComponent.tsx:318]: Its the default value assigned. We take it as ok to be able to use the alpha.

Comment (deltakosh) [reply inspector/src/components/actionTabs/tabs/propertyGrids/materials/textures/textureEditorComponent.tsx:317]: ok sounds good! thanks!

**Pull request merged**